src/tests/JIT/Delegate/Tests.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Not sure what the right location for these tests is.

Reply:
@SamMonoRT @lambdageek any ideas?

Reply:
Not sure how/when we run these test  - https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/mono/mono/tests/delegate.cs

I'm fine leaving it in the same location as in this PR, but would rename Tests.csproj and Tests.cs to a better name like virtual delegate test or similar ?

Reply:
Added @trylek for any suggestions on right place to add the new test in the PR ?

Reply:
I'm not aware of any strict guidance regarding test placement, in fact the entire test tree is a huge compilation with many legacy aspects. I see there is a folder <code>JIT/Methodical/delegate</code> that only contains a single test so I'm wondering whether that might be sufficient so that we don't create another one-test folder but I don't think it's a huge deal.

Reply:
This could be a library test in `src/libraries/System.Runtime/tests/System/DelegteTests`.  There's a `class CreateDelegateTests` in there.

But actually this looks like a pretty common scenario: what are we testing here.  `Delegate.CreateDelegate` works in mono, doesn't it?  Is it the fact that `ToString` is virtual?

Reply:
@lambdageek  Actually this test scenario is failing for FullAOT mode since invoke_callvirt wrapper can not be generated at compile time...so this is why I've included it in this PR

Reply:
> @lambdageek Actually this test scenario is failing for FullAOT mode since invoke_callvirt wrapper can not be generated at compile time...so this is why I've included it in this PR

I see.  Pretty good test, then :smile: 

I like Tomas' suggestion of JIT/Methodical/delegate

Reply:
After all, this might ultimately improve code coverage even for CoreCLR, that's the beauty of cross-pollination ;-)